prov/efa/src/rxr/rxr.h

```diff
@@ -357,6 +357,7 @@ struct rxr_rx_entry {
 
 	size_t iov_count;
 	struct iovec iov[RXR_IOV_LIMIT];
+	void *desc[RXR_IOV_LIMIT];
```

Comment:
Can you comment this item so we know what it's for?

Reply:
Will do.